src/python/grpcio_tests/tests/unit/_6891_test.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests regression of issue 6891."""
```

Comment:
Can we name tests for what they actually test?  In this case probably something like early_ok or consume_some_stream_messages, and just reference the bug in a comment?

Reply:
Sure. Done.